shop-local/build/asset-manifest.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Files from /shop-local/build
should not be added to GitHub as they are built using npm and file names will change everything the build is done.